lib/kadalulib.py

```diff
@@ -93,7 +94,10 @@ def is_server_pod_reachable(hosts, port=24007, timeout=20):
         retry_count = 0
         while retry_count < 4:
             try:
-                sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+                if netaddr.valid_ipv(host):
```

Comment:
This looks like a typo. netaddr.valid_ipv ??

Reply:
netaddr package has this method to validate ipv4, it's not a typo

Reply:
Could you point it out to me?

https://netaddr.readthedocs.io/en/latest/api.html